homeassistant/components/websocket_api/connection.py

```diff
@@ -93,11 +93,11 @@ def async_handle_exception(self, msg, err):
             err_message = 'Unauthorized'
         elif isinstance(err, vol.Invalid):
             code = const.ERR_INVALID_FORMAT
-            err_message = 'Invalid format'
+            err_message = str(err)
```

Comment:
We should use `humanize_error` on the exception.

Reply:
And don't forget a test 👍

Reply:
changed to `humanize_error` and added test